tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -152,6 +158,19 @@ pub fn command(s: &str) -> Result<Command, cli::Error> {
                         Ok(Command::Schema)
                     });
 
+    let sync_parser = string(SYNC_COMMAND)
+                    .with(spaces())
+                    .with(arguments())
+                    .map(|args| {
+                        if args.len() < 1 {
+                            bail!(cli::ErrorKind::CommandParse("Missing required argument".to_string()));
+                        }
+                        if args.len() > 2 {
```

Comment:
I think this can probably be simplified — just have the parser only take one argument?